### ISSUE TITLE
formas flexionadas

### ISSUE THREAD

**Issue by arademaker** [own-pt/openWordnet-PT#140]
já li algumas coisas, vou me manifestar em outras poucas, o tempo está  mesmo curto.
Quanto ao gênero, assim como no caso do número (vésperas/véspera), o número de exceções é maior que imaginamos.
Ok para brasileira/brasileiro, mas o que fazer com os casos abaixo, como garantir que a regra irá uniformizar o que não deve?
bolsa-bolso
poça-poço
barraca-barraco
etc

Por isso, minha sugestão é que, no caso dos N, tenhamos sempre as duas formas - feminino e masculino. E nisso estamos apoiados em nomes relevantes da descrição linguística do português, como Mattoso Camara, que diz que substantivos têm gênero próprio, e apenas adjetivos se flexionam em gênero. (e brasileira/brasileiro pode ser tanto N como ADJ).

Quanto ao número, o Houaiss tem uma entrada para recursos:
ª recursos
n substantivo masculino plural
6    aptidões naturais; dons, talentos, dotes
Ex.: r. estilísticos 
7    meios pecuniários, bens materiais; posses, riquezas
Ex.: ficou sem r. até para o aluguel 
8    Derivação: sentido figurado.
riquezas, fundos, meios de que se pode dispor
Exs.: r. humanos
 r. naturais

**Comment by vcvpaiva** [comment]
>E nisso estamos apoiados em nomes relevantes da descrição linguística do português, como Mattoso Camara, que diz que substantivos têm gênero próprio, e apenas adjetivos se flexionam em gênero. 

referencia, please?

me parece que os outros wordnets de linguas latinas nao fazem desse jeito nao. so' listam as excecoes, como as que vc listou acima.

**Comment by arademaker** [comment]
Limpando inbox achei estas mensagens antigas e estou apenas registrando. A mensagem foi da Claudia, acabamos não convergindo para nada e não sei se concordo com a idéia não. Mas temos ainda algumas coisas para revisar, então o issue.

**Comment by vcvpaiva** [comment]
e com relacao a numero @livyreal  estava fazendo um trabalho de marar os substantivos que teem que ser plural em portugues.

Livy Real, Valeria de Paiva. Plurality in Wordnets. ArXiv, 01 July 2016. Proceedings of the LexSem+Logics Workshop 2016, associated with PROPOR 2016. [PDF]

**Comment by vcvpaiva** [comment]
sim, eu notei q voce estava lendo emails antigas, estou complemento o esforco arqueologico.

**Comment by livyreal** [comment]
Mattoso Camara Jr. Estrutura da língua portuguesa (1970).

O melhor seria termos a possibilidade de marcar gênero e número como features e não colocando todas as formas possíveis da palavra. Ainda que o Mattoso diga que "menino" e "menina" são palavras distintas, não é assim que as gramáticas tradicionais tratam o fenômeno e a generalização das gramáticas tradicionais é menos custosa do que a do Mattoso, conversa melhor com os lematizadores para português e não nos obriga a decidir o que é nome e o que é adjetivo (algo que é mesmo indecidível sem considerar contexto).
